config-linux.md

```diff
@@ -742,6 +742,8 @@ If `intelRdt` is not set, the runtime MUST NOT manipulate any `resctrl` pseudo-f
 The following parameters can be specified for the container:
 
 * **`closID`** *(string, OPTIONAL)* - specifies the identity for RDT Class of Service (CLOS).
+  As a special case, value `.` means that the container should be assigned to the default CLOS (the
```

Comment:
would '/' be more explicit?  The value '.' looks to me like the expectation is the "current CLOS"

Reply:
I'm fine with this, too. I was struggling with the name myself